### ISSUE TITLE
KCom2 - Ability to rename images on products

### ISSUE THREAD

**Issue by stephen-fueldesign** [Kademi/kademi-dev#6699]
waterware have 1700 products. with 2-3 images each.
Some have been uploaded as SKU.png, SKU.2.jpg SKU.3.jpg etc.
Other sections, images uploaded as SKU.1.png SKU.2.jpg etc.

Which gives me issues with $format.sortByProperty() as SKU.png or SKU.1.png give different results.

But if Kademi could even allow ability to set a default or primary image in a set, then this may resolve issues altogether too, then client to upload images per product, and choose their default image. Problem solved too.

Please can this be set as an important feature upgrade.

Thanks

**Comment by bradmac** [comment]
> then client to upload images per product, and choose their default image

So the client would have to manually choose the default image for 1700 products ... is that practical?

**Comment by bradmac** [comment]
The intention is that users set the image order by uploading them with appropriate file names. 

Would that be easier then manually editing every product?

**Comment by stephen-fueldesign** [comment]
You're right. I think it's best to just be able to rename a product, in case client wants to re-order (shuffle) images about. and in this instance, fix the 300ish broken first names. They effectively need to remove the .1 in a few of their images and everything will be good to go.

**Comment by stephen-fueldesign** [comment]
I think setting a primary image is a fail. ignore that request.

**Comment by bradmac** [comment]
As discussed, we can add a rename feature, that would be perfectly reasonable. But it probably wouldnt help the client. I think it would be best for the client to maintain their own 'source of truth' and simply upload it when they need to make changes.

**Comment by stephen-fueldesign** [comment]
I think setting a primary image is a fail. ignore that request.

**Comment by stephen-fueldesign** [comment]
@bradmac Sorry for the confusion, but can we still get "Ability to rename images on products."
This was a two part request, and I just cancelled the 2nd part. But the first request remains.
Thanks!

**Comment by bradmac** [comment]
OK, edited to reflect the change you want.

**Comment by bradmac** [comment]
scheduled for p0
